### PR TITLE
Added special output (dev feature) for FindOpcodeInFilesForm.

### PR DESCRIPTION
**Pull request by Mag-nus**
Added turbine chat decoders.
Cleaned up some of the code in Form1.cs
Added F3 to go to next highlighted row.

ProcessFile() in FindOpcodeInFilesForm has some stub code with samples, on how you can search many pcaps at once and output results into the second tab.

### CODE REVIEW

File: aclogview/CM_Communication.cs [hunk 1]
```diff
@@ -140,6 +146,53 @@ public override void contributeToTreeView(TreeView treeView)
...
+        public uint AllegianceChannel;
+        public uint GeneralChannel;
+        public uint TradeChannel;
+        public uint LFGChannel;
+        public uint RoleplayChannel;
+        public uint Olthoi;
+        public uint Society;
+        public uint Unknown_3;
+        public uint Unknown_4;
+        public uint Unknown_5;
```
Comment (tfarley) [inline aclogview/CM_Communication.cs]: FYI these unknowns seem to be SocietyCelHan, SocietyEldWeb, and SocietyRadBlo - each corresponding to their respective society.

File: aclogview/Form1.cs [hunk 2]
```diff
@@ -780,54 +766,78 @@ class ListViewItemComparer : IComparer<ListViewItem> {
...
 
-        private void treeView_ParsedData_AfterSelect(object sender, TreeViewEventArgs e) {
-            updateText();
+            loadPcap(openFile.FileName);
+        }
+
+
+        private void mnuItem_EditNextHighlightedRow_Click(object sender, EventArgs e)
+        {
+            for (int i = listView_Packets.TopItem.Index + 1; i < listView_Packets.Items.Count; i++)
```
Comment (tfarley) [inline aclogview/Form1.cs]: If you add a quick null-check here on TopItem, you can prevent the app from crashing when no packets are loaded and you hit F3.

**Comment by tfarley** [comment]
Might it be helpful to add a previous-highlighted-row option as well? Maybe bound to F2 or something? Totally up to you though.

**Comment by Mag-nus** [comment]
Should be good now.

**Pull request merged**